config/gatekeeper/kustomization.yaml

```diff
@@ -43,6 +43,13 @@ patches:
   target:
     kind: Deployment
     name: gatekeeper-audit
+- patch: |-
+    - op: replace
+      path: /spec/template/spec/containers/0/resources/limits/memory
+      value: 1Gi
+  target:
+    kind: Deployment
+    name: gatekeeper-audit
```

Comment:
All the patches above this one target the `gatekeeper-audit` Deployment. Can we consolidate them into a single `patch` array?